busypie/condition.py

```diff
@@ -47,6 +48,10 @@ def with_description(self, description: str) -> 'ConditionBuilder':
         self._condition.description = description
         return self._new_builder_with_cloned_condition()
 
+    def at_least(self, at_least: int) -> 'ConditionBuilder':
```

Comment:
You don't need to write an actual function, you're all set with ```_time_property_evaluator_for```

Reply:
fixed